templates/internet-of-things/smart-displays.html

```diff
@@ -0,0 +1,337 @@
+{% extends "internet-of-things/base_things.html" %}
+
+
+{% block title %}Smart displays | Ubuntu for the Internet of Things {% endblock %}
+{% block meta_description %}With hundreds of thousands of devices deployed the world over, Ubuntu Core and Ubuntu Frame are the perfect bundle for deploying robust, reliable and secure digital signage solutions and kiosks.{% endblock meta_description %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1fFvBl-ZZvKDFV6OMv7O4kudQKtyxfvSU60rfVdKluAQ/edit{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-strip--image is-dark is-deep u-no-padding--top u-no-padding--bottom" style="background-image: url(https://assets.ubuntu.com/v1/2c3ad93c-digital-signage-2.png); background-position: right bottom; background-size: 1246px auto;">
+  <div class="p-strip--suru-half-and-half-reversed">
+    <div class="row u-equal-height u-vertically-center">
+      <div class="col-6">
+        <h1>
+          Digital signage and kiosk <br class="u-hide--large">with Ubuntu
+        </h1>
```

Comment:
"The easy, secure, and reliable way of building smart displays for your business." is missing

```suggestion
        </h1>
        <p class="p-heading--4">The easy, secure, and reliable way of building smart displays for your business.</p>
```